### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'parentNode') when reloading task view

### ISSUE THREAD

**Issue by Freundschaft** [go-vikunja/frontend#123]
### Description

When performing a full reload (e.g. with chrome dev tools active), the task view fails to load with the following message:

```
TypeError: Cannot read properties of undefined (reading 'parentNode')
    at parentNode (runtime-dom.esm-bundler.js:33:30)
    at ReactiveEffect.Kt [as fn] (runtime-core.esm-bundler.js:5777:11)
    at ReactiveEffect.run (reactivity.esm-bundler.js:178:19)
    at Ge.qe.update (runtime-core.esm-bundler.js:5814:51)
    at At (runtime-core.esm-bundler.js:5641:18)
    at Ue (runtime-core.esm-bundler.js:5576:7)
    at De (runtime-core.esm-bundler.js:5040:11)
    at Ve (runtime-core.esm-bundler.js:5427:7)
    at Fe (runtime-core.esm-bundler.js:5319:7)
    at Le (runtime-core.esm-bundler.js:5167:7)
logError @ runtime-core.esm-bundler.js:226

```

Going back to board view and navigating to the task view again helps fix the issue, somehow this only happens with chrome dev  tools open.

### Vikunja Frontend Version

0.21.0

### Vikunja API Version

0.21.0

### Browser and version

chrome 116.0.5845.110

### Can you reproduce the bug on the Vikunja demo site?

No

**Comment by kolaente** [comment]
Which view were you on when that happend? How did you get there? (which pages did you visit before etc)

**Comment by kolaente** [comment]
If you can reproduce it reliably, can you enable sentry and try again? (`window.SENTRY_ENABLED` in the `index.html` file or using the `VIKUNJA_SENTRY_ENABLED` with the docker image)

**Comment by Freundschaft** [comment]
yes just did this :) thanks!

**Comment by Freundschaft** [comment]
yes, so I do the following:

1. go to board in kanban view
2. open task
3. ctrl shift r reload, after some times the error will occur

**Comment by Typhonragewind** [comment]
I'm having a similar issue, i believe. Whenever i open a task that contains an image embedded in the description, i can't get out of it without reloading the whole page, and the console gives me these errors:
![image](https://github.com/go-vikunja/frontend/assets/7875917/b07dda1f-a409-4880-8e8a-c5a16f64c970)

Currently on Frontend and API v0.22.0 via Docker

**Comment by kolaente** [comment]
This is now fixed in [0b3604d167](https://kolaente.dev/vikunja/frontend/commit/8ea97f3ffc16eb795badf623bd195cd9b8d9a388). Please check with the latest unstable build (on [try](https://try.vikunja.io)) to see if the problem went away.
